src/rtpp_log_stand.c

```diff
@@ -248,7 +248,7 @@ _rtpp_log_write_va(struct rtpp_log_inst *rli, int level, const char *function,
     fprintf(stderr, rli->format_se[0], rtpp_time_buff, strlvl(level),
       call_id, function);
     vfprintf(stderr, format, ap);
-    fprintf(stderr, rli->format_se[1]);
+    fprintf(stderr, "%s", rli->format_se[1]);
```

Comment:
This won't work. The rli->format_se[] cotains *actual% format with %foo.

Reply:
Hello @sobomax 

Doesn't rli->format_se[1] contain an actual string? In line 110 I see

`rli->format_se[1] = "\n";`

I'm not a programmer, but that looks like a string to me, not a format specifier.

Kind regards,
Seb

Reply:
Ah, ok, you are right! Thanks and sorry for the confusion.. Merged.